src/app/withdraw/withdraw.component.ts

```diff
@@ -68,7 +68,10 @@ export class WithdrawComponent implements OnInit {
             switchMap(() => this.popupService.new({
                 type: 'success',
                 title: 'Transaction signed',
-                text: 'Transaction is being processed...'
+                text: 'Transaction is being processed...',
+                customClass: 'popup-success',
+                position: 'top',
+                confirmButtonText: 'Continue <i class="fas fa-arrow-right ml-3"></i>'
```

Comment:
Even though this is OK, I would avoid mixing HTML inside typescript files. If this arrow could be some UTF8 icon, it would be nice.